app/assets/stylesheets/application.css

```diff
@@ -10,7 +10,7 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require uswds
```

Comment:
Renamed because I couldn't figure out a way to get the USWDS compiler to generate the file with its original name.